source/_integrations/androidtv_remote.markdown

```diff
@@ -44,6 +43,132 @@ Examples of URLs to pass as activity for some popular apps:
 | Prime Video | https://app.primevideo.com
 | Disney+ | https://www.disneyplus.com
 
+Examples:
+
+```yaml
+# Launch the Netflix app
+service: media_player.play_media
+data:
+  media_content_type: url
+  media_content_id: https://www.netflix.com/title
```

Comment:
These should have been quoted, see our YAML styling guides on our developer documentation for more information.

Please adjust this in a new PR (same for all other examples).

Reply:
Done in the new PR: https://github.com/home-assistant/home-assistant.io/pull/27308